sklearn/preprocessing/tests/test_data.py

```diff
@@ -812,6 +812,18 @@ def test_robust_scaler_iris():
     assert_array_almost_equal(iqr, 1)
 
 
+def test_robust_scaler_iris_quantiles():
+    X = iris.data
+    scaler = RobustScaler(quantile_low=10, quantile_high=90)
+    X_trans = scaler.fit_transform(X)
+    assert_array_almost_equal(np.median(X_trans, axis=0), 0)
+    X_trans_inv = scaler.inverse_transform(X_trans)
+    assert_array_almost_equal(X, X_trans_inv)
+    q = np.percentile(X_trans, q=(10, 90), axis=0)
+    iqr = q[1] - q[0]
+    assert_array_almost_equal(iqr, 1)
+
+
```

Comment:
Is it expected to raise at `__init__` time, at `set_params` time or at `fit` time?